### DISCUSSION TITLE
Should CIE 1976 and CAM16-UCS methods give equal results?

### DISCUSSION THREAD

**Discussion by senyai** [colour-science/colour#765] [Q&A] [answered]
```python
>>> from colour import  delta_E
>>> a = [100.00000000, 21.57210357, 272.22819350]
>>> b = [100.00000000, 426.67945353, 72.39590835]
>>> delta_E(a, b, method='CIE 1976')
451.71330197359117
>>> delta_E(a, b, method='CAM16-UCS')
451.71330197359117
```

**Comment by thomasmansencal** [comment, marked as answer]
Hi @senyai,

I know it is confusing but both methods are indeed using Euclidean distance under the hood, however, the inputs should be different: You cannot use a Lab colour with CAM16-UCS but a `J'a'b'` array from the CAM16 CAM:

```
    Returns the difference :math:`\\Delta E_{ab}` between two given
    *CIE L\\*a\\*b\\** or :math:`J'a'b'` colourspace arrays using given method.
```

At some point, I will roll the power values of Huang et al. (2015) as described in #370 and the result will be different even with the same input.

**Comment by thomasmansencal** [comment]
Closing this one for now @senyai, cheers :)
